app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb

```diff
@@ -0,0 +1,111 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow < ManageIQ::Providers::AnsiblePlaybookWorkflow
+  def load_transitions
+    self.state ||= 'initialize'
+
+    {
+      :initializing      => {'initialize'         => 'waiting_to_start'},
+      :start             => {'waiting_to_start'   => 'pre_execute'},
+      :pre_execute       => {'pre_execute'        => 'execute'},
+      :execute           => {'execute'            => 'running'},
+      :poll_runner       => {'running'            => 'running'},
+      :post_execute      => {'running'            => 'post_execute_poll'},
+      :post_execute_poll => {'post_execute_poll'  => 'post_execute_poll'},
+      :finish            => {'*'                  => 'finished'},
+      :abort_job         => {'*'                  => 'aborting'},
+      :cancel            => {'*'                  => 'canceling'},
+      :error             => {'*'                  => '*'}
+    }
+  end
+
+  def post_execute
```

Comment:
If you're going to overwrite the post_execute method you should still call `cleanup_git_repository` which is done by the core method

Reply:
easiest way is to do your own thing and call `super`

Reply:
@chessbyte Hmm, yeah, especially if the base class implementation ever get's changed in the future.

Reply:
The issue with that is the `post_execute` method in core calls `queue_signal(:finish)` at the end where here he needs to `queue_signal(:post_execute_poll)`

Reply:
good catch

Reply:
I've changed according to Adam's suggestion